### PR TITLE
Feature/lesson 2

### PR DESCRIPTION
**Pull request by MashaLoban**

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ghost) [inline about/index.html]: Please change the title.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <br/>
+    <p><h2>Потому, что мне кажется это <em>интересным.</em></h2></p>
+    <br/>
+    <p><h3>Потому, что мне кажется это <mark>перспективным.</mark></h3></p>
+    <br/>
+    <p><h4>Потому, что в случае, если мне  будет интересно программирование, и это будет приносить <ins>доход</ins>, то я буду немножко счастливее, чем сейчас.<br/> Т.к. буду знать, что у меня получилось добиться результата. <del>Но это не точно.</del> И немножко добавим (H<sub>2</sub>O)<sup>2</sup>.</h4></p>
+    <br/>
+    <a href="mailto:predel666@mail.ru">	&#9993; Моя почта для связи </a>
+    <br/>
+    <a href="https://github.com/MashaLoban" target="Профиль">&#128293; А это мой профиль на Github</a>
```
Comment (ghost) [inline about/index.html]: Seems the target attribute should have another value.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Почему я выбрала направление Front-end разработки?</h1>
+    <br/>
+    <hr>
+    <br/>
+    <p><q>Суть жизни – найти самого себя.</q><strong> Мухаммад Икбал</strong></p>
+    <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (ghost) [inline index.html]: Please change the title.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+    <h1>Моя первая страница</h1>
+    <p><a href="../if-html/about/index.html">Об авторе</a></p>
```
Comment (ghost) [inline index.html]: Seems the value for href attribut is wrong.
```suggestion
    <p><a href="./about/index.html">Об авторе</a></p>
```

File: about/index.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    <br/>
+    <p><h2>Потому, что мне кажется это <em>интересным.</em></h2></p>
+    <br/>
+    <p><h3>Потому, что мне кажется это <mark>перспективным.</mark></h3></p>
+    <br/>
+    <p><h4>Потому, что в случае, если мне  будет интересно программирование, и это будет приносить <ins>доход</ins>, то я буду немножко счастливее, чем сейчас.<br/> Т.к. буду знать, что у меня получилось добиться результата. <del>Но это не точно.</del> И немножко добавим (H<sub>2</sub>O)<sup>2</sup>.</h4></p>
+    <br/>
+    <a href="mailto:predel666@mail.ru">	&#9993; Моя почта для связи </a>
```
Comment (ghost) [inline about/index.html]: Please delete empty space.

File: about/index.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Почему я выбрала направление Front-end разработки?</h1>
+    <br/>
+    <hr>
```
Comment (ghost) [inline about/index.html]: Make a closing tag.

File: about/index.html (hunk 1 above)
Comment (MashaLoban) [reply about/index.html]: The title changed.

File: index.html (hunk 4 above)
Comment (MashaLoban) [reply index.html]: The title changed

File: index.html (hunk 5 above)
Comment (MashaLoban) [reply index.html]: fixed

File: about/index.html (hunk 3 above)
Comment (MashaLoban) [reply about/index.html]: fixed

File: about/index.html (hunk 6 above)
Comment (MashaLoban) [reply about/index.html]: done

File: about/index.html (hunk 7 above)
Comment (MashaLoban) [reply about/index.html]: done

File: about/index.html (hunk 2 above)
Comment (MashaLoban) [reply about/index.html]: fixed

**Comment by ghost** [comment]
Good job.

**Pull request merged**